acquia-pipelines.yml

```diff
@@ -0,0 +1,44 @@
+# This is an example Acquia Pipelines configuration file. For complete
+# documentation and other examples, see https://docs.acquia.com/acquia-cloud/develop/pipelines/yaml/
+#
+# If you'd like a project which sets up testing and pipelines for you, consider
+# using Acquia BLT: https://github.com/acquia/blt
+version: 1.1.0
+services:
+  - mysql
+  - php:
+      version: 7.1
+
+events:
+  build:
+    steps:
+    # Build a development codebase suitable for testing.
+    # Uncomment these lines if you have setup dev dependencies and testing for
+    # your project.
+    #  - build-dev:
+    #      type: script
+    #      script:
+    #        - composer validate --no-check-all --ansi
+    #        - composer install
+    #
+    # Install application.
+    # If you have tests defined for your application, uncomment these lines
+    # to install and test. (Note that you will also need to uncomment the
+    # "build-dev" step above so that you have a codebase to work with.) This
+    # assumes you have a well-configured settings.php file committed to your
+    # repo and all testing dependencies like Behat and PHPUnit as dev
+    # dependencies.
+    #  - test:
+    #      type: script
+    #      script:
+    #        - drush site-install
+    #        - drush runserver --default-server=builtin 8080 &>/dev/null &
+    #        - behat --stop-on-failure
+    #        - phpunit --configuration ./PATH_TO_YOUR_CONFIG --group YOUR_TEST_TAGS
```

Comment:
This gives the impression like the --group and --configuration options are required for PHPUnit. Is that an impression we want to give? Maybe just call 'phpunit' and assume they have the configuration set up as they want?